Sming/SmingCore/Data/HttpHeaders.cpp

```diff
@@ -0,0 +1,51 @@
+/****

```

Comment:
A better location of the HttpHeaders.* files will be under `Sming/SmingCore/Network/Http/`. Please, move them there.